.dockerignore

```diff
@@ -1,4 +1,5 @@
 .git
+**/node_modules
```

Comment:
does this really make a difference? For me all node_modules folders were ignored before - this doesn't work the same way as .gitignore works?

Reply:
I've done some trials, here's what I found:

- with only `node_modules`, I CTRL-C'ed the Docker build, and the context was over 500mb
- with only `**/node_modules`: context reaches 295 mb
- with both `node_modules` and `**/node_modules`: context reaches 384 kb

Still, according [to the documentation](https://docs.docker.com/engine/reference/builder/#dockerignore-file), I would have thought that only `**/node_modules` was necessary, but it seems like we also need `node_modules`

Reply:
Interesting, so this works totally differently than gitignore, I did not know, thank you for clarifying!